packages/vitest-pool-workers/test/console.test.ts

```diff
@@ -42,10 +41,10 @@ test.skipIf(process.platform === "win32")(
 		`,
 		});
 		await waitFor(() => {
-			expect(result.stdout).toMatch("stdout | index.test.ts:2:9\nnew global");
 			expect(result.stdout).toMatch(
-				"stdout | index.test.ts:4:10\nnew describe"
```

Comment:
This seems to be a change in the Vitest output to remove line numbers (verified by upgrading Wrangler to Vitest v2+ and seeing the change)